README.md

```diff
@@ -1,10 +1,70 @@
 # Drizzle Utils
 
-A library of frontend utilities for making dapps
+A library of tools for interacting with Ethereum smart contracts with a focus on [RxJS](https://github.com/ReactiveX/rxjs) streams so you can easily subscribe to events, contract state, and read-only methods.
```

Comment:
Perhaps we should talk about how Drizzle Utils fits into the greater hierarchy of things along with Drizzle and DRC. I don't thing it's clear to a new user which one to use.

Reply:
@honestbonsai I'm trying to shorten this as much as possible. I want this to be a one-sentence description, any ideas?

Reply:
@honestbonsai I've updated the description. Is it any better? Also, I think having "Drizzle" in the name is what might be confusing, but maybe we should just leave it for now. We can always rebrand after we get feedback from our first "soft release" to Truffle/ConsenSys.

Reply:
Yea it's probably the "Drizzle" part of the name. I'm fine with rebranding after the soft release.